cfg/qt.cfg

```diff
@@ -3276,6 +3531,27 @@
       <function name="back" yields="item"/>
     </access>
   </container>
+  <container id="qtVector" startPattern="QVector &lt;" inherits="qtContainer" opLessAllowed="true">
+    <type string="std-like"/>
```

Comment:
@danmar I wonder what this configuration is for. I just copied it from `QList` since `QVector` is similar to it. IMHO the name suggests that it specifies that this container works like `std::string`, but I have no idea. There is no comment for this configuration in the source code and looking at where this is referenced I am still not sure what it does.
If it specifies that this container is string-like then I guess this configuration is wrong for `QList` and `QVector` configuration. For `QString` it could make sense. But I am not sure if it makes sense for `QStringList` where it is also specified.
In std.cfg there is also `<type associative="std-like"/>`. I have no idea when to use that too.

Reply:
> In std.cfg there is also `<type associative="std-like"/>`. I have no idea when to use that 

I guess it marks these containers as associative like it is explained in https://en.wikipedia.org/wiki/Associative_containers. `QMap` likely could be marked with `<type associative="std-like"/>` i guess. And others too: "Qt also provides these associative containers: QMap, QMultiMap, QHash, QMultiHash, and QSet." (from https://doc.qt.io/qt-5/containers.html)

Reply:
I removed the std-like string configuration for QVector. I am somewhat sure now that it does not belong here.